src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -0,0 +1,132 @@
+import QtQuick 2.14
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+
+Item {
+    id: statusBaseInput
+
+    property bool multiline: false
+
+    property bool clearable: false
+
+    property alias inputMethodHints: edit.inputMethodHints
+
+    property alias selectedText: edit.selectedText
+    property alias selectedTextColor: edit.selectedTextColor
+    property alias selectionStart: edit.selectionStart
+    property alias selectionEnd: edit.selectionEnd
+
+    property alias color: edit.color
+    property alias font: edit.font
+    property alias verticalAlignmet: edit.verticalAlignment
+    property alias horizontalAlignment: edit.horizontalAlignment
+
+    property alias placeholderText: placeholder.text
+    property alias placeholderTextColor: placeholder.color
+    property alias placeholderFont: placeholder.font
+
+    property real leftPadding: 16
+    property real rightPadding: 16
+    property real topPadding: 11
+    property real bottomPadding: 11
+
+    implicitWidth: 200
+    implicitHeight: multiline ? Math.max(edit.implicitHeight, 40) : 40
+
+    clip: true
+
+    Rectangle {
+        anchors.fill: parent
+        color: Theme.palette.baseColor2
+        radius: 8
+    }
+
+    Flickable {
```

Comment:
we should probably add ScrollBars since we have set `wrapMode: TextEdit.NoWrap`.